src/frontend-pwa/src/views/Report/Report.tsx

```diff
@@ -1,12 +1,131 @@
-import { TextField } from '@mui/material';
+/* eslint-disable react/no-array-index-key */
+/**
+ * TODO:    - implement error handling
+ *          - make less ugly
+ *
+ * @summary - Event reporting page for the Wayfinder application.
+ *          - Users can submit reports about events they witness.
+ *
+ * @author  TylerMaloney
+ */
+
+import React, { useState, useEffect } from 'react';
+import { Button } from '../../components/Button/Button';
+import { StyledReportContainer, StyledReportOuterDiv, StyledTextArea } from './report.styles';
+
+export function Report() {
+  const [eventType, setEventType] = useState('');
+  const [details, setDetails] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
+  const [latitude, setLatitude] = useState(0);
+  const [longitude, setLongitude] = useState(0);
+  const eventOptions: Array<string> = ['Damaged Infrastructure', 'Animal Sighting', 'Suggestion/Complaint', 'Miscellaneous'];
+
+  const handleEventTypeChange = (e: { target: { value: React.SetStateAction<string>; }; }) => {
+    setEventType(e.target.value);
+  };
+
+  const handleDetailsChange = (e: { target: { value: React.SetStateAction<string>; }; }) => {
+    setDetails(e.target.value);
+  };
+
+  const handlePhoneNumberChange = (e: { target: { value: React.SetStateAction<string>; }; }) => {
+    setPhoneNumber(e.target.value);
+  };
+
+  const handleSubmit = async (e: { preventDefault: () => void; }) => {
+    e.preventDefault();
+
+    const currentTime = new Date();
+
+    const formData = {
+      latitude,
+      longitude,
+      eventType,
+      details,
+      phoneNumber,
+      time: currentTime,
+    };
+
+    try {
+      // Send JSON object to the API endpoint
+      const response = await fetch('https://wf-api-ec1236-dev.apps.silver.devops.gov.bc.ca/api/report', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(formData),
+      });
+
+      if (response.ok) {
+        // Reset form fields
+        setEventType('');
+        setDetails('');
+        setPhoneNumber('');
+      } else {
+        throw new Error('Form data submission failed');
+      }
+    } catch (error) {
+      throw new Error('fetch failed');
```

Comment:
throw error, to catch error, to throw different error?

Reply:
Ya looks like you only need to catch the error and console log it. You shouldn't be throwing an error in catch